database/schema.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE endentities(
+      id          SERIAL PRIMARY KEY,
+      label       VARCHAR NOT NULL,
+      signer_id   VARCHAR NOT NULL,
+      is_current  BOOLEAN NOT NULL,
+      x5u         VARCHAR NULL,
+      created_at  TIMESTAMP WITH TIME ZONE NOT NULL
+);
+CREATE INDEX endentities_latest_idx ON endentities(label, signer_id, is_current);
+ALTER TABLE endentities ADD CONSTRAINT endentities_unique_label UNIQUE (label);
+
+CREATE ROLE myautographdbuser;
+ALTER ROLE myautographdbuser WITH NOSUPERUSER INHERIT NOCREATEROLE NOCREATEDB LOGIN PASSWORD 'myautographdbpassword';
+GRANT SELECT, UPDATE, INSERT ON endentities TO myautographdbuser;

```

Comment:
we're using UPDATE to set the x5u and is_current? I wonder if we can drop that and have an effective audit log.

Reply:
I'll try. it might be possible by reorganizing some of the functions.

Reply:
done in https://github.com/mozilla-services/autograph/pull/208/commits/00543342f25a35f02ff55b5e9fadbcbf35c55628